### DISCUSSION TITLE
Is it possible to configure bruno to skip TLS verification? The same way the -k flag does for curl?

### DISCUSSION THREAD

**Discussion by Olian04** [usebruno/bruno#564] [Q&A] [answered]
I have a use case where I need to disable TLS verification. Using curl i can simply add the -k flag. How can I configure bruno to do the same?

**Comment by helloanoop** [comment, marked as answer]
Have you tried disabling ssl in the preferences.. You can open preferences by clicking on the gear icon on the left bottom.

**Reply by Olian04** [reply]
that did it! Didn't realize the cog at the bottom left and the cog at the top right were for different menus.

**Reply by Olian04** [reply]
@helloanoop There should should probably be an override for this option on each request. Especially if the point "Copy Curl directly into Bruno" from the roadmap, is to be implemented 1-to-1 with how curl works.
